### ISSUE TITLE
Unable to start vike

### ISSUE THREAD

**Issue by yaseenisolated** [vikejs/vike#1317]
### Description
```
pnpm vite
The CJS build of Vite's Node API is deprecated. See https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated for more details.
2:03:09 PM [vike][Warning] We recommend setting /Users/yaseenhamdulay/stitch/src/Administration/package.json#type to "module", see https://vike.dev/CJS
2:03:09 PM [vike@0.4.148][config][Bug] You stumbled upon a bug in Vike's source code. Go to https://github.com/vikejs/vike/issues/new and copy-paste this error; a maintainer will fix the bug (usually under 24 hours).
    at crawlPlusFiles (/Users/yaseenhamdulay/stitch/node_modules/.pnpm/vike@0.4.148_vite@5.0.5/node_modules/vike/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getVikeConfig/crawlPlusFiles.js:18:27)
    at findPlusFiles (/Users/yaseenhamdulay/stitch/node_modules/.pnpm/vike@0.4.148_vite@5.0.5/node_modules/vike/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getVikeConfig.js:803:64)
    at loadInterfaceFiles (/Users/yaseenhamdulay/stitch/node_modules/.pnpm/vike@0.4.148_vite@5.0.5/node_modules/vike/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getVikeConfig.js:78:29)
    at loadVikeConfig (/Users/yaseenhamdulay/stitch/node_modules/.pnpm/vike@0.4.148_vite@5.0.5/node_modules/vike/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getVikeConfig.js:238:46)
    at loadVikeConfig_withErrorHandling (/Users/yaseenhamdulay/stitch/node_modules/.pnpm/vike@0.4.148_vite@5.0.5/node_modules/vike/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getVikeConfig.js:201:21)
    at getVikeConfig (/Users/yaseenhamdulay/stitch/node_modules/.pnpm/vike@0.4.148_vite@5.0.5/node_modules/vike/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getVikeConfig.js:72:29)
    at getConfigVikPromise (/Users/yaseenhamdulay/stitch/node_modules/.pnpm/vike@0.4.148_vite@5.0.5/node_modules/vike/dist/cjs/node/plugin/plugins/config/index.js:33:97)
  VITE v5.0.5  ready in 555 ms
  ➜  Local:   http://localhost:3000/
  ➜  Network: use --host to expose
  ➜  press h + enter to show help
^[[A2:03:17 PM [vike][request(1)] HTTP request: /
2:03:17 PM [vike][request(1)] Couldn't load configuration: see error above.
^C
```

my vite.config.js looks like
```
import react from '@vitejs/plugin-react';
import vike from 'vike/plugin'
import { defineConfig } from 'vite';

// https://vitejs.dev/config/
export default defineConfig({
    plugins: [react(), vike()],
    root: './src/client', // Set the root to the directory containing your index.html
    build: {
        // This allows using CJS shared packages
        commonjsOptions: {
            include: [/shared/, /node_modules/],
        },
        outDir: '../../dist', // Adjust the output directory as needed
    },
});
```

I'm also trying to serve Vike using koa with a script like
```

async function setupVite(app: Koa) {
    const router = new Router();

    const isProduction = process.env.NODE_ENV === 'production';
    const root = `${__dirname}/..`;

    if (isProduction) {
        app.use(sirv(`${root}/dist/client`));
    } else {
        const viteDevMiddleware = (await createViteServer({
            root,
            server: { middlewareMode: true },
        })).middlewares;

        app.use(connect(viteDevMiddleware));
    }

    router.get('(.*)', async (ctx, next) => {
        const pageContextInit = { urlOriginal: ctx.originalUrl };
        const pageContext = await renderPage(pageContextInit);
        const { httpResponse } = pageContext;
        if (!httpResponse) return next();

        const { body, statusCode, contentType } = httpResponse;
        ctx.status = statusCode;
        ctx.type = contentType;
        ctx.body = body;
    });

    app.use(router.routes());
}
(async () => {
    await setupVite(app);
    app.listen(settings.port);
    logger.info('Initialized 🚀');
})().catch(console.error)

```

**Comment by brillout** [comment]
Also, insert the following after the line `13` of `/Users/yaseenhamdulay/stitch/node_modules/.pnpm/vike@0.4.148_vite@5.0.5/node_modules/vike/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getVikeConfig/crawlPlusFiles.js` and report back the log result:

```diff
async function crawlPlusFiles(userRootDir, outDirAbsoluteFilesystem, isDev) {
+  console.log('userRootDir', userRootDir);
+  console.log('outDirAbsoluteFilesystem', outDirAbsoluteFilesystem);
    (0, utils_js_1.assertPosixPath)(userRootDir);
```

**Comment by yaseenisolated** [comment]
```
pnpm vite
The CJS build of Vite's Node API is deprecated. See https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated for more details.
3:16:04 PM [vike][Warning] We recommend setting /Users/yaseenhamdulay/stitch/src/Administration/package.json#type to "module", see https://vike.dev/CJS
userRootDir /Users/yaseenhamdulay/stitch/src/Administration/src/client
outDirAbsoluteFilesystem /Users/yaseenhamdulay/stitch/src/Administration/src/client/../../dist/
3:16:04 PM [vike@0.4.148][config][Bug] You stumbled upon a bug in Vike's source code. Go to https://github.com/vikejs/vike/issues/new and copy-paste this error; a maintainer will fix the bug (usually under 24 hours).
    at crawlPlusFiles (/Users/yaseenhamdulay/stitch/node_modules/.pnpm/vike@0.4.148_vite@5.0.5/node_modules/vike/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getVikeConfig/crawlPlusFiles.js:20:27)
    at findPlusFiles (/Users/yaseenhamdulay/stitch/node_modules/.pnpm/vike@0.4.148_vite@5.0.5/node_modules/vike/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getVikeConfig.js:803:64)
    at loadInterfaceFiles (/Users/yaseenhamdulay/stitch/node_modules/.pnpm/vike@0.4.148_vite@5.0.5/node_modules/vike/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getVikeConfig.js:78:29)
    at loadVikeConfig (/Users/yaseenhamdulay/stitch/node_modules/.pnpm/vike@0.4.148_vite@5.0.5/node_modules/vike/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getVikeConfig.js:238:46)
    at loadVikeConfig_withErrorHandling (/Users/yaseenhamdulay/stitch/node_modules/.pnpm/vike@0.4.148_vite@5.0.5/node_modules/vike/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getVikeConfig.js:201:21)
    at getVikeConfig (/Users/yaseenhamdulay/stitch/node_modules/.pnpm/vike@0.4.148_vite@5.0.5/node_modules/vike/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getVikeConfig.js:72:29)
    at getConfigVikPromise (/Users/yaseenhamdulay/stitch/node_modules/.pnpm/vike@0.4.148_vite@5.0.5/node_modules/vike/dist/cjs/node/plugin/plugins/config/index.js:33:97)
```

**Comment by yaseenisolated** [comment]
Yup, looks like `vike@0.4.147` works just fine so this is likely a new regression.

**Comment by brillout** [comment]
Fix pre-released in `0.4.148-commit-70e7518`.
